src/web/core/src/main/java/org/geoserver/web/services/BaseServiceAdminPage.java

```diff
@@ -123,8 +123,14 @@ void init(final IModel<T> infoModel) {
         }
 
         form.add(onlineResource);
-        form.add(new CheckBox("enabled"));
-        form.add(new CheckBox("citeCompliant"));
+        CheckBox enabled = new CheckBox("enabled");
+        enabled.setOutputMarkupId(true);
+        enabled.setMarkupId("enabled");
+        form.add(enabled);
```

Comment:
Using this to get the generated html to have the correct id (for label association). There is probably more of this to be done....